From: Simon McVittie <smcv@debian.org>
Subject: pick up cpp flags for dbus-1 from pkg-config instead of hard-coding
 Hard-coding the include path for dbus-1 is not recommended, and was broken
 by the multiarch transition. The interface provided by upstream can be
 summarized as "use pkg-config".
Bug-Debian: http://bugs.debian.org/634451
Bug-Ubuntu: https://launchpad.net/bugs/749278

diff --git a/src/hd/Makefile b/src/hd/Makefile
index 3aa4819..be5b346 100644
--- a/src/hd/Makefile
+++ b/src/hd/Makefile
@@ -4,7 +4,7 @@ CLEANFILES	= version.h
 
 include $(TOPDIR)/Makefile.common
 
-CFLAGS		+= -I /usr/include/dbus-1.0 -I /usr/lib64/dbus-1.0/include -I /usr/lib/dbus-1.0/include
+CFLAGS		+= $(shell pkg-config --cflags dbus-1)
 
 version.h: $(TOPDIR)/VERSION
 	@echo "#define HD_VERSION_STRING \"`cat $(TOPDIR)/VERSION`\"" >$@
